uncharted-aske/HMI
# Load BGraph Node/Edge Files from S3

This PR contains a package change. Please run `yarn install`

To test that the change works:
- Open the bio view and check your network tab. Ensure that the bgraph files are being transferred from S3 successfully.
